### ISSUE TITLE
Repo throwing a notice

### ISSUE THREAD

**Issue by banderon** [Automattic/vip-go-mu-plugins#1617]
A member of our team reported that, after pulling down the latest version of this repo, they started getting a notice. I pulled in the latest version for myself and started getting the notice as well. The issue looks to be with the two-factor plugin, triggered after Query Monitor.

I've tried wiping everything and setting up a new VM/WP install from scratch, but am still getting the notice.

<img width="1386" alt="Screen Shot 2020-06-09 at 13 25 25" src="https://user-images.githubusercontent.com/533529/84180640-d45c9200-aa55-11ea-8a48-9ed6e10ba6cb.png">

Removing `two-factor.php` kills the notice.

**Comment by kadamwhite** [comment]
Bump on this; a doing_it_wrong is being fired because `wp_register_script` is being called on the wrong hook, causing errors to be output throughout the admin and frontend if WP_DEBUG is true. This is a pretty invasive error, and while we can temporarily disable it locally by filtering doing_it_wrong to suppress any messages about `wp_register_script`, it would be strongly preferable to have the plugin updated to not trigger this error. Thanks for any assistance you can provide!

**Comment by mjangda** [comment]
The upstream fix is being discussed here: https://github.com/WordPress/two-factor/pull/356

**Comment by joemcgill** [comment]
Looks like a fix for this has been merged upstream.

Currently getting around this locally using the following:

```php
add_filter( 'wpcom_vip_enable_two_factor', function () {
	return defined( 'VIP_GO_ENV' ) && VIP_GO_ENV;
} );
```

**Comment by paulgibbs** [comment]
I'm seeing this as well. Please could the update be included into here?

**Comment by rinatkhaziev** [comment]
Folks, you've probably noticed it's been fixed, but for posterity, this has been resolved via #1737
